components/templates/Profile.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+import AuthorCollection from '../../interfaces/AuthorCollection'
+import { MarkdownFileWithUrl } from '../../pages/authors'
+import ContentCardList from '../atoms/ContentCardList'
+import PageLayout from '../layouts/PageLayout'
+
+export interface Props {
+ author: AuthorCollection
+ records: MarkdownFileWithUrl[]
+}
+
+const Author: React.FunctionComponent<Props> = props => {
+  const authorName = props.author.name
```

Comment:
Does this need to be a variable?

Reply:
It doesn't but we use it in more than one place that's why it is currently a variable. What's your suggestion?

Reply:
We use it in two places, which seems fine to me. So I would use `props.author.name` in those places.

Reply:
I think so because we use it in more than one place